dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/PhoneConfirmation.cshtml.cs

```diff
@@ -57,4 +81,14 @@ public override void OnPageHandlerExecuting(PageHandlerExecutingContext context)
             context.Result = new RedirectResult(_linkGenerator.RegisterPhone());
         }
     }
+
+    private string NormaliseMobileNumber(string mobileNumber)
+    {
+        return new string(RemoveUkCountryCode(mobileNumber).Where(char.IsDigit).ToArray()).TrimStart('0');
+    }
+
+    private string RemoveUkCountryCode(string mobileNumber)
+    {
+        return Regex.Replace(mobileNumber, @"^(\+44|0044)", "");
```

Comment:
For national numbers user can enter either 07877 722 722 or 00447877 722 722 and we want to treat as equal. Not the case for international numbers